Translation licoes-modelo.md

joanacvp opened the pull request:
I just made the translation to portuguese of the page "Programming Historian English Language Lesson Template". @aracele can you please just check if everything is ok considering the translation and the compatibility to brazilian portuguese? 
Also, @rivaquiroga can you check if everything is ok with the things I have inserted?
Thank you very much for your help

joanacvp committed:
Update licoes-modelo.md

aracele committed:
Update licoes-modelo.md

Everything seems good to me, I've just fixed a typo.

aracele reviewed (dismissed):
@joanacvp It seems good to me, I've just fixed a small typo and now I think it's good to go :)

rivaquiroga commented on pt/licoes-modelo.md:
```diff
@@ -1,131 +1,130 @@
+
+**Apague tudo o que está acima desta linha quando estiver pronto a submeter a lição**.
+
+título: O SEU TÍTULO AQUI
```
@joanacvp, the fields in the YAML header should not be translated. Only what is currently in uppercase has to be translated to portuguese.

joanacvp committed:
Update licoes-modelo.md

@rivaquiroga I apologize the confusion. Is it ok like this? Thank you very much for your help

DanielAlvesLABDH commented:
Hi @rivaquiroga. Sorry but I really don't understand why that part of the file has to be in English. Or am I seeing the wrong file? I thought @joanacvp had done something similar to the Spanish version of the file: https://programminghistorian.org/es/plantilla-leccion.md Did we miss something here? Thanks!

rivaquiroga commented:
> Hi @rivaquiroga. Sorry but I really don't understand why that part of the file has to be in English. Or am I seeing the wrong file? I thought @joanacvp had done something similar to the Spanish version of the file: https://programminghistorian.org/es/plantilla-leccion.md Did we miss something here? Thanks!

In the Spanish version of the template the YAML header fields are not translated. They are not translated either in the PT lesson that is already published: https://raw.githubusercontent.com/programminghistorian/jekyll/gh-pages/pt/licoes/preservar-os-seus-dados-de-investigacao.md 
Those fields are common to all the lessons. When a lesson is published, you see them in the correct language thanks to the translations you provided on the `snippets.yml` file.

DanielAlvesLABDH commented:
Thanks @rivaquiroga probably my confusion is about where are those YAML fields in the file. In line 19 we should keep "título" as "title" and then translate the rest? Sorry again for the confusion. Always learning...

rivaquiroga commented:
Yes, from line 20 to 45. I just saw that @joanacvp made the changes.

DanielAlvesLABDH commented:
Ok @rivaquiroga, I understand now. But the text "Delete everything above this line when ready to submit your lesson." and all the above we can translate, correct?

rivaquiroga commented:
@DanielAlvesLABDH, yes, that part can be translated.

DanielAlvesLABDH committed:
Update licoes-modelo.md

Voltei a colocar o texto em Português, com excepção das linhas indicadas pela Riva
